check_tags.sh

```diff
@@ -3,17 +3,20 @@
 
 set -e
 
-if [[ -z "${GITHUB_TOKEN}" ]]; then
+if [[ -z "${GH_TOKEN}" ]] && [[ -z "${GITHUB_TOKEN}" ]] && [[ -z "${GH_ENTERPRISE_TOKEN}" ]] && [[ -z "${GITHUB_ENTERPRISE_TOKEN}" ]]; then
```

Comment:
Is it missing the `else` statement like in `update_version.sh`?

Reply:
It was. Changed it to match.